utilities/src/main/java/io/syndesis/qe/utils/dballoc/DBAllocationManager.java

```diff
@@ -0,0 +1,95 @@
+package io.syndesis.qe.utils.dballoc;
+
+import javax.ws.rs.client.Client;
+import javax.ws.rs.client.Invocation;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+import java.io.IOException;
+import java.io.Reader;
+import java.io.StringReader;
+import java.util.Map;
+import java.util.Properties;
+
+import io.syndesis.qe.utils.RestUtils;
+
+public class DBAllocationManager {
+
+    private static final String URL_GUI = "http://dballocator.mw.lab.eng.bos.redhat.com:8080/Allocator/AllocatorServlet";
+
+    private static final String REQUESTEE = "syndesis-qe";
+
+    private static final int EXPIRE = 60;
+
+    private static final Client client = RestUtils.getInsecureClient();
+
+    public static DBAllocation allocate(String dbLabel) {
+        return allocate(dbLabel, REQUESTEE, EXPIRE);
+    }
+
+    private static DBAllocation allocate(String label, String requestee, int expire) {
+
+        DBAllocation alloc = new DBAllocation(label);
+        Invocation.Builder invocation = createAllocateInvocation(label, requestee, expire);
+
+        final Response response = invocation.get();
+
+        final String text = response.readEntity(String.class);
+
+        try {
+            alloc.setAllocationMap(loadProps(text));
+            alloc.setUuid(alloc.getAllocationMap().get("uuid"));
+            return alloc;
+        } catch (IOException e) {
+            e.printStackTrace();
+            return null;
```

Comment:
Is it correct behaviour to return null or is it better if we threw Assertions.fail here?